app/forms.py

```diff
@@ -225,12 +225,6 @@ def clean_mobile_number(self):
         if not mobile_number:
             return
 
-        if mobile_number.startswith('1'):
-            RegexValidator('^\d{11}$', message=_("Enter a valid mobile number in "
-               "the form 14034228916"))(mobile_number)
-        else:
-            RegexValidator('^\d{10}$', message=_("Enter a valid mobile number in "
-                   "the form 14034228916"))(mobile_number)
```

Comment:
Does the localflavor library handle giving a logical response to the user (I presume that's what this code is for)?
And would it respond in french for the french site?